boss-ws/src/main/java/org/genomebridge/boss/http/resources/ObjectResource.java

```diff
@@ -76,11 +76,24 @@ public CopyResponse resolveForCopy(@PathParam("objectId") String objectId,
     @DELETE
     public String delete(@PathParam("objectId") String objectId,
                          @HeaderParam(REMOTE_USER_HEADER) String userName) {
-        ErrorDesc err = api.deleteObject(objectId,userName);
+        ErrorDesc err = api.deleteObject(objectId, userName);
         if ( err != null )
             throwWAE(err);
         return objectId;
     }
 
+    @Path("multi")
+    @Produces("application/json")
+    @POST
+    public ResolveResponse getUploadID(@PathParam("objectId") String objectId,
+                                       @HeaderParam(REMOTE_USER_HEADER) String userName) {
+        ResolveResponse resp = new ResolveResponse();
```

Comment:
Since we're returning just the URI, let's reuse the CopyResponse class instead of ResolveResponse.